source/docs/advanced/features/external-logins.md

```diff
@@ -0,0 +1,314 @@
+  ],
+...
+```
+
+## Magento1 Requirements
+
+For the external logins to work with the Magento1 module in addition to [Adding the necessary modules](#Adding-the-necessary-modules) you need to ensure you are have the magento1 connector on 1.4.2 or later. 
+
+## Magento2 Requirements
+
+For the external logins to work with the Magento2 module in addition to [Adding the necessary modules](#Adding-the-necessary-modules) you need to ensure you are have the magento1 connector on 2.5.2 or later. 
+
+## Ensuring the login buttons show
+
+The `auth-external-login` registered above comes with an override to the `AdditionalLoginFormActions` component. The `AdditionalLoginFormActions` component was added to the `LoginForm` in 2.13.0 so if you overrode `LoginForm` please refer to the [migration guides](/docs/appendices/migration-guides#Login-Form-Update) for more details of how to add it to your `LoginForm`. You might also want to add the `FlashMessages` component to your `LoginForm` so that authentication errors show to the user ([migration guides](/docs/appendices/migration-guides#Login-Form-Update)).
+
+After ensuring that `AdditionalLoginFormActions` and `FlashMessages` are in your `LoginForm` the "Sign in with Facebook" and "Sign in with Google" buttons should now appear in your login page/modal.
+
+## Adding the stylesheet
+
+For the out of the box login buttons you can include the following stylesheet in your `main.scss` file:
+
+```diff
+...
+@import "~theme/modules/modules";
+@import "~theme/pages/pages";
++@import "~theme/external-login.scss";
+...
+```
+
+## Configuring the buttons
+
+To configure the buttons shown for external logins you can override the `theme/modules/User/LoginForm/AdditionalLoginFormActions/AdditionalLoginFormActions.js` file and add or remove the buttons you want.
+
+## Configuring the Facebook provider
+
+The `FacebookProvider` needs the `clientId` and the `clientSecret` to work. You can get these from your app on the [facebook developer console](https://developers.facebook.com/apps/).
+
+To configure the `clientId` and the `clientSecret` you need to define the folowing environment variables:
+* `FRONT_COMMERCE_FACEBOOK_CLIENT_ID` for the `clientId`
+* `FRONT_COMMERCE_FACEBOOK_CLIENT_SECRET` for the `clientSecret`
+
+### Configuring Redirect URIs
+
+You need to configure the allowed redirect URIs on your [facebook developer console](https://developers.facebook.com/apps/). To do so go you need to select your app and then go to `Facebook Login > Settings` section on the left menu. There you will find a field called `Valid OAuth Redirect URIs`. You need to fill this field with your stores URLs folowed by `/external-login/facebook/callback`. For example if you have 2 stores `https://example.com/en` and `https://example.com/fr`, you need to put `https://example.com/en/external-login/facebook/callback` and `https://example.com/fr/external-login/facebook/callback` in the `Valid OAuth Redirect URIs` field.
```

Comment:
the nuance here is:
* needing to register all the stores
* postfix the stores urls by `/external-login/facebook/callback`

these they cant find in facebook docs

Reply:
Maybe this would be clearer then ?

```suggestion
You need to configure the allowed redirect URIs on your [facebook developer console](https://developers.facebook.com/apps/). 

Note that the `Valid OAuth Redirect URIs` should contain all your stores URLs followed by `/external-login/facebook/callback`. 
 
For example for stores `https://example.com/en` and `https://example.com/fr`, you need to put `https://example.com/en/external-login/facebook/callback` and `https://example.com/fr/external-login/facebook/callback` in the `Valid OAuth Redirect URIs` field.
```

Reply:
that works thx

Reply:
also updated the google one in the same spirit